app/models/user.rb

```diff
@@ -1,2 +1,11 @@
 class User < ApplicationRecord
+  has_many :posts
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+
+  def increment_posts_counter
+    update(posts_counter: posts_counter + 1)
+  end
 end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, the relationships shown in the ERD are not complete. 

![image](https://github.com/emarkees/Gist_Blog/assets/77069011/9b52373e-760c-44ff-a6ae-cab4cbebc54a)

A `User` has many `Posts`, `Comments` and `Likes`. You only specified `Posts`. Please consider fixing this! 🤓 🔧 

- [ ] Also, the `increment_posts_counter` function should be in your `Post` model. According to the requirements:

![image](https://github.com/emarkees/Gist_Blog/assets/77069011/4949d39d-313b-4f00-91ea-bd1dd1483f45)